charmcraft/commands/store/store.py

```diff
@@ -251,6 +251,11 @@ def whoami(self):
         )
         return result
 
+    @_store_client_wrapper(auto_login=False)
```

Comment:
possibly we don't need this decorator, as this method is called internally by public methods that have the decorator already (looks like the other private methods don't use the decorator).

Reply:
Oh, it should be wrapped already.